aten/src/ATen/native/quantized/cudnn/Conv.cpp

```diff
@@ -199,22 +199,28 @@ at::SmallVector<int64_t, 4> MakeConvOutputShape<2>(
 // the parameter quantized_output is a quantized tensor
 template <int kSpatialDim>
 template <bool kReluFused>
-void PackedConvWeightCudnn<kSpatialDim>::apply_impl_helper(const at::Tensor& quantized_output, const at::Tensor& input,
-                                                           double bias_multiplier, double requantize_multiplier) {
+void PackedConvWeightCudnn<kSpatialDim>::apply_impl_helper(const at::Tensor& quantized_output, const at::Tensor& input, double output_scale) {
   if (quantized_output.numel() == 0) {
     return;
   }
   at::Tensor conv_output = at::empty(quantized_output.sizes(), at::device(at::kCUDA).dtype(at::kFloat), at::MemoryFormat::ChannelsLast);
   // TODO: combine empty & fill_ using full_like or full
   at::Tensor requantize_multiplier_tensor = at::empty(quantized_output.sizes(), at::device(at::kCUDA).dtype(at::kFloat), at::MemoryFormat::ChannelsLast);
+  auto act_scale = input.q_scale();
+  auto weight_scale = orig_weight.q_scale();
```

Comment:
`orig_weight` --> `orig_weight_`

Reply:
hmm looks like my rebase yesterday wasn't done properly. I'll fix this